--- test_train.py
+++ test_train.py
@@ -1,7 +1,7 @@
-class Model(nn.Module):
+class FMD(nn.Module):
 
     def __init__(self):
-        super(Model, self).__init__()
+        super(FMD, self).__init__()
         self.features = self._make_layers(cfg, batch_norm=True)
         self.classifier = nn.Sequential(
             nn.Linear(8 * 15 * 10, 512),
@@ -18,7 +18,8 @@
         x = self.features(x)
         x = x.view(x.size(0), -1)
         x = self.classifier(x)
-        return x 
+        return x
+
     def _make_layers(self,cfg,batch_norm):
         layers = []
         in_channels = 3